test/JDBC/upgrade/latest/schedule

```diff
@@ -327,3 +327,4 @@ BABEL_GRANT_CONNECT
 BABEL-sp_helpdb
 BABEL-2795
 babelfish_integrity_checker
+BABEL_SCHEMATA
```

Comment:
latest to latest only does MVU. This also need minor version test, add to 14.5 to latest or similar

Reply:
Alright, added